### ISSUE TITLE
Requesting image size fails

### ISSUE THREAD

**Issue by nielsmaerten** [nielsmaerten/peppermint#9]
Looks like whatever is passed into the function when getting the size for a new image, is not its url:

Stackdriver Error Reporting error detail

Error: You should provide an URI string or a "request" options object.
https://console.cloud.google.com/errors/1794925657192919263?time=P30D&project=peppermint-wallpapers

**Comment by nielsmaerten** [comment]
Full stacktrace:

```
Error: You should provide an URI string or a "request" options object.
    at requestImageSize (/user_code/node_modules/peppermint/node_modules/request-image-size/index.js:25:27)
    at getImageProperties (/user_code/node_modules/peppermint/dist/peppermint.umd.js:150:12)
    at /user_code/node_modules/peppermint/dist/peppermint.umd.js:173:28
    at next (native)
    at /user_code/node_modules/peppermint/dist/peppermint.umd.js:158:71
    at __awaiter$2 (/user_code/node_modules/peppermint/dist/peppermint.umd.js:154:12)
    at Function.onNewMasterImage (/user_code/node_modules/peppermint/dist/peppermint.umd.js:172:35)
    at exports.newMasterImage.functions.database.ref.onCreate.event (/user_code/index.js:14:29)
    at /user_code/node_modules/firebase-functions/lib/cloud-functions.js:54:20
    at process._tickDomainCallback (internal/process/next_tick.js:135:7)
```

**Comment by nielsmaerten** [comment]
Fixed in 0735c62
